src/core/dbus-execute.c

```diff
@@ -1775,20 +2146,43 @@ int bus_exec_context_set_transient_property(
 
                 return 1;
 
-        } else if (streq(name, "SELinuxContext")) {
+        } else if (STR_IN_SET(name, "SELinuxContext", "AppArmorProfile", "SmackProcessLabel")) {
+                bool ignore;
                 const char *s;
 
-                r = sd_bus_message_read(message, "s", &s);
+                r = sd_bus_message_enter_container(message, 'r', "bs");
```

Comment:
OK, I will update so. Then, should we change `SD_BUS_PROPERTY()` to "s" to much `bus_exec_context_set_transient_property()`, or should not?